test/python/test_mapper_coupling.py

```diff
@@ -23,13 +23,13 @@ def test_empty_coupling_class(self):
 
     def test_coupling_str(self):
         coupling_list = [(0, 1), (0, 2), (1, 2)]
```

Comment:
can we make this a list of lists.

Reply:
See b2954fbc41ac093bb9ac1fb3a971ded626989dc8.